test/integration/cloudresource_e2e_test.go

```diff
@@ -86,14 +87,16 @@ var _ = Describe(fmt.Sprintf("%s,%s: NetworkPolicy On Cloud Resources", focusAws
 				utils.CollectSupportBundle(kubeCtl, path.Join(supportBundleDir, fileName))
 			}
 
-			nss := []*v1.Namespace{namespace, otherNamespace}
-			for _, ns := range nss {
-				if ns == nil {
-					continue
+			if !withAgent {
```

Comment:
why are we not deleting ns in agent case?

Reply:
Because deleting and recreating ns takes time. We are using the same ns for agented tests so no need to delete and recreate and wait for it.

Reply:
Could you add a comment why you dont prefer to delete ns?